docs/020-environment.md

```diff
@@ -12,6 +12,10 @@ This token should be added to the environment variables as `GITHUB_TOKEN`.
 Here is how to do it with Travis CI:
 [Defining Variables in Repository Settings](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings).
 
+If you don't want to have a globaly available github token on your env, you
```

Comment:
This is probably fine, but I'd probably word it as something like:
>Alternatively, you can provide the GitHub token in a file. GoReleaser will check `~/.config/goreleaser/github_token` by default.

I would also recommend documenting the `env_files` config setting somewhere.